static/js/tools/map/info.tsx

```diff
@@ -44,15 +44,15 @@ export function Info(): JSX.Element {
           <b>Max Temperature Relative To 2006, RCP 4.5</b> across counties in{" "}
           <a
             href={
```

Comment:
Maybe add a webdriver test for the map landing page link, can just click one (first) link and confirm chart shows up. Maybe similar for scatter plot.

Reply:
done